libkpod/logs.go

```diff
@@ -0,0 +1,64 @@
+package libkpod
+
+import (
+	"fmt"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/hpcloud/tail"
+)
+
+const (
+	// LogsBaseDir specifies the default directory in which logs are stored
+	logsBaseDir = "/var/log/containers"
+)
+
+// LogOptions contains all of the options for displaying logs in kpod
+type LogOptions struct {
+	Details   bool
+	Follow    bool
+	SinceTime time.Time
+	Tail      int64
+}
+
+// GetLogs gets each line of a log file and, if it matches the criteria in logOptions, sends it down logChan
+func GetLogs(server *ContainerServer, container string, logChan chan string, opts LogOptions) error {
```

Comment:
nit: Can we make this func (server *ContainerServer) GetLogs(...) instead? I'd prefer to make our public API methods of ContainerServer to the greatest extent possible.